runner/test/cases/server/plugins/engine/datepartsfield.test.ts

```diff
@@ -67,7 +67,7 @@ suite("Date parts field", () => {
         {
           path: "approximate__day",
           href: "#approximate__day",
-          name: "approximate__day",
+          name: "myComponent__approximate__day",
```

Comment:
Can myComponent have a more descriptive name (if it's required)?

Reply:
I mean its just a test really. The already existing test data was named myComponent so i just went with that